gulpfile.coffee

```diff
@@ -19,7 +21,7 @@ handle = (stream) ->
     do stream.end
 
 test_files = 'test/tests/*.coffee'
-coffee_files = 'assets/js/**/*'
+coffee_files = 'assets/**/*.coffee'
```

Comment:
I disagree because I'm also using "base: plugins" and it's useful to have the literal path side by side (I tried both ways)